lib/file/widgets/file_tile.dart

```diff
@@ -1,24 +1,22 @@
 import 'package:flutter/material.dart';
+import 'package:pedantic/pedantic.dart';
```

Comment:
Maybe we should also export `package:pedantic/pedantic.dart` from `package:schulcloud/app/app.dart`? Seems like `unawaited` is generally useful.